Client/Frontend/Browser/BackForwardListViewController.swift

```diff
@@ -122,34 +125,82 @@ class BackForwardListViewController: UIViewController, UITableViewDataSource, UI
         guard let bvc = self.bvc else {
             return
         }
-        tableView.snp_updateConstraints { make in
-            make.bottom.equalTo(self.view).offset(bvc.shouldShowFooterForTraitCollection(newCollection) ? -UIConstants.ToolbarHeight : 0)
+        if bvc.shouldShowFooterForTraitCollection(newCollection) != snappedToBottom {
+            self.dismissing = true
```

Comment:
This was a holdover from something else I was trying. I'll remove it — but we might as well remove that variable entirely, as it never seems to be used.